client/app/components/QueryBasedParameterInput.jsx

```diff
@@ -89,7 +89,7 @@ export default class QueryBasedParameterInput extends React.Component {
           value={this.state.value}
           onChange={onSelect}
           options={map(options, ({ value, name }) => ({ label: String(name), value }))}
-          optionFilterProp="children"
+          optionFilterProp={optionFilterProp || "children"}
```

Comment:
Why the `optionFilterProp` change instead of leaving it to be replaced from `otherProps`? If you are using it it shouldn't be an "unwanted" prop 😅